src/partials/sections/testimonials-carousel.hbs

```diff
@@ -0,0 +1,80 @@
+<section class="testimonials-carousel mt-4 py-4">
+  <div class="container text-center">
+    <h2 class="h1 mb-0">As seen in</h2>
+    <div id="testimonialsCarousel" class="carousel slide" data-ride="carousel">
+
+      <div class="carousel-inner">
+        <div class="carousel-item active">
```

Comment:
quote icons as on LIVE are missing, please integrate them by using the SVG version (ask to designers the SVG)